component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticatorConstants.java

```diff
@@ -106,4 +106,7 @@
 
     public static final String SCREEN_VALUE = "&screenValue=";
     public static final String SHOW_EMAIL_ADDRESS_IN_UI = "showEmailAddressInUI";
+    public static final String USE_EVENT_HANDLER_BASED_EMAIL_SENDER = "useEventHandlerBasedEmailSender";
+    public static final String TEMPLATE_TYPE = "TEMPLATE_TYPE";
+    public static final String EVENT_NAME = "EmailOTP";
 }
```

Comment:
Add new line here

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/26/commits/4c8a1a700c541009035ba290999b9609af6e95d2